pyfreenas/pool.py

```diff
@@ -0,0 +1,146 @@
+from enum import Enum, unique
+from typing import TypeVar
+
+TMachine = TypeVar("TMachine", bound="Machine")
+TPoolStatus = TypeVar("TType", bound="PoolStatus")
+# TPoolScanState = TypeVar("TType", bound="PoolScanState")
+# TPoolEncryptionAlgorithm = TypeVar("TType", bound="PoolEncryptionAlgorithm")
+
+
+@unique
+class PoolStatus(Enum):
+    ONLINE = "ONLINE"
+    # FIXME: What are the other statuses?
```

Comment:
I imagine `OFFLINE` (too many disk missing) and `DEGRADED` (missing a disk, but not fully offline) are states.  Can you create some test `zvol`s to make a test pool, and then remove some of them to check?

Reply:
Added stated OFFLINE and DEGRADED. I think there could be others. Must look more at documentation.